dbt/adapters/snowflake/relation.py

```diff
@@ -11,7 +11,7 @@ class SnowflakeRelation(DefaultRelation):
         'quote_policy': {
             'database': False,
             'schema': False,
-            'identifier': False,
+            'identifier': True,
```

Comment:
should we quote both schema and identifier here? Feels weird to do one and not the other